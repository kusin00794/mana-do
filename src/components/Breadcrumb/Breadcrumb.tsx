import React from 'react';
import { Link } from 'react-router-dom';

import RoutesString from '../../pages/routesString';

const Breadcrumb = () => {
  return (
    <ul className="nav">
      <li className="nav-item">
        <Link to={RoutesString.Welcome} className="nav-text">
          <i className="fa fa-home" />
        </Link>
      </li>
    </ul>
  );
};

export default Breadcrumb;
