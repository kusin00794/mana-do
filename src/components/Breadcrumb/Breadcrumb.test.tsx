import React from 'react';
import { BrowserRouter } from 'react-router-dom';

import { testSnapshots } from '../../utils/test';
import Breadcrumb from './Breadcrumb';

const Wrapper = (props: any) => (
  <BrowserRouter>
    <Breadcrumb {...props} />
  </BrowserRouter>
);
describe('Breadcrumb Page', () => {
  testSnapshots(Wrapper, [
    {
      props: {},
      description: 'render Breadcrumb Page',
    },
  ]);
});
