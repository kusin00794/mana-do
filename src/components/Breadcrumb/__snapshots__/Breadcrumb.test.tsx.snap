// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb Page snapshots render Breadcrumb Page 1`] = `
<ul
  className="nav"
>
  <li
    className="nav-item"
  >
    <a
      className="nav-text"
      href="/"
      onClick={[Function]}
    >
      <i
        className="fa fa-home"
      />
    </a>
  </li>
</ul>
`;
